### PR TITLE
feat: holesky support

### PR DESCRIPTION
**Pull request by rkrasiuk**
## Description

Add support for [Holesky](https://github.com/eth-clients/holesky) testnet.

Blocked by https://github.com/gakonst/ethers-rs/pull/2569

### CODE REVIEW

File: crates/primitives/src/chain/spec.rs [hunk 1]
```diff
@@ -157,6 +157,49 @@ pub static SEPOLIA: Lazy<Arc<ChainSpec>> = Lazy::new(|| {
...
+            (Hardfork::Byzantium, ForkCondition::Block(0)),
+            (Hardfork::Constantinople, ForkCondition::Block(0)),
+            (Hardfork::Petersburg, ForkCondition::Block(0)),
+            (Hardfork::Istanbul, ForkCondition::Block(0)),
+            (Hardfork::MuirGlacier, ForkCondition::Block(0)),
+            (Hardfork::Berlin, ForkCondition::Block(0)),
+            (Hardfork::London, ForkCondition::Block(0)),
+            (
+                Hardfork::Paris,
+                ForkCondition::TTD { fork_block: None, total_difficulty: U256::ZERO },
```
Comment (rkrasiuk) [inline crates/primitives/src/chain/spec.rs:187]: note: add a test that checks that paris hardfork is activated correctly at genesis

**Comment by Rjected** [comment]
unblocked now that https://github.com/gakonst/ethers-rs/pull/2569 is merged?

**Review by mattsse — APPROVED** [review]
lgtm

**Review by gakonst — APPROVED** [review]
send it

**Pull request merged**